test/test-installer-script.mjs

```
@@ -27,203 +30,124 @@ function pass(message) {
-  console.log('Python Installer Script Test');
+  console.log('UV Installer Flow Test');
   console.log('='.repeat(60));
   console.log();
 
-  const coreDir = path.join(os.homedir(), '.platformio');
-  const penvDir = path.join(coreDir, 'penv');
+  const testVenvDir = path.join(os.tmpdir(), `pio-test-venv-${Date.now()}`);
+  let uvExe;
 
+  // Test 1: Find uv
+  console.log('Test 1: Finding uv executable...');
```

ESLint

[error] Unexpected console statement.

(no-console)